### PR TITLE
node-conformance-pr-2017-12-07

### PR DESCRIPTION
**Pull request by lichuqiang**
The original translation was posted to website repo directly, without syncing to the k8smeetup repo, so make a synchronize now.

### CODE REVIEW

**Comment by lichuqiang** [comment]
Synchronize from the website, approve no-issue

File: docs/admin/node-conformance.md [hunk 1]
```diff
@@ -1,100 +1,195 @@
...
 
+<!--
 ## Node Conformance Test
 
 *Node conformance test* is a containerized test framework that provides a system
 verification and functionality test for a node. The test validates whether the
 node meets the minimum requirements for Kubernetes; a node that passes the test
 is qualified to join a Kubernetes cluster.
+-->
+## 节点合规性测试
```
Comment (ScorpioCPH) [inline docs/admin/node-conformance.md:27]: `一致性测试` maybe better.

File: docs/admin/node-conformance.md [hunk 2]
```diff
@@ -1,100 +1,195 @@
...
 
 * The test leaves some docker images on the node, including the node conformance
   test image and images of containers used in the functionality
   test.
 * The test leaves dead containers on the node. These containers are created
   during the functionality test.
+-->
+## 注意事项
+
+* 测试会在节点上遗留一些Docker镜像， 包括节点合规性测试本身的镜像，和功能测试相关的镜像。
```
Comment (ScorpioCPH) [inline docs/admin/node-conformance.md:194]: nit: whitespace between Chinese characters and English characters.

File: docs/admin/node-conformance.md (hunk 1 above)
Comment (lichuqiang) [reply docs/admin/node-conformance.md:27]: 参考该 [comments](https://github.com/kubernetes/website/pull/5564#discussion_r145010453)

Comment (ScorpioCPH) [reply docs/admin/node-conformance.md:27]: Sorry, don't noticed that but i don't think so, just FYI:

http://www.myzaker.com/article/5a0c0df71bc8e0f32800000e/
http://www.techweb.com.cn/news/2017-11-15/2606693.shtml
http://cloud.chinabyte.com/1/14349501.shtml
http://www.sdnlab.com/20127.html

I think `软件一致性认证` is a common sense in china and maybe better to understand.

**Pull request merged**